.github/workflows/build.yml

```diff
@@ -0,0 +1,64 @@
+name: Build and Release CuteCapture
+on:
+  push:
+    branches: [master]
+  pull_request:
+    branches: [master]
+jobs:
+  linux:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v1
+      - name: Build
+        run: |
+          sudo apt-get install -y libusb-1.0.0-dev libsfml-dev
+          ./autogen.sh && ./configure && make
+          chmod +x Cute3DSCapture
+          # TODO: make binary statically linked as well (until then needs libusb+libsfml installed)
```

Comment:
I think I baked an option for this into the makefile previously. Maybe we could make use of that here.

Reply:
There's a `--with-staticsfml`, but I need the `.a` files which don't seem to be in the Ubuntu repo:
https://github.com/vgmoose/Cute3DSCapture/runs/1267358124?check_suite_focus=true
> make: *** No rule to make target '/usr/local/lib/libsfml-graphics-s.a', needed by 'Cute3DSCapture'.  Stop.
379

If I manually build SFML with `cmake -DBUILD_SHARED_LIBS:BOOLEAN=false .`, I do get the `.a` files, and I can install and point to them, however I get this error then when trying to build:
```
g++  -std=c++11   -o Cute3DSCapture main.o 3dscapture.o /usr/local/lib/libsfml-graphics-s.a /usr/local/lib/libsfml-window-s.a /usr/local/lib/libsfml-system-s.a -lusb-1.0 -lGL -ljpeg -lfreetype -lpthread -ludev -lxcb-image -lxcb-randr -lxcb -lX11-xcb -lX11 -lsfml-graphics-s -lsfml-window-s -lsfml-system-s 
/usr/bin/ld: /usr/local/lib/libsfml-window-s.a(VulkanImplX11.cpp.o): undefined reference to symbol 'dlclose@@GLIBC_2.2.5'
/usr/bin/ld: /lib/x86_64-linux-gnu/libdl.so.2: error adding symbols: DSO missing from command line
collect2: error: ld returned 1 exit status
make: *** [Makefile:381: Cute3DSCapture] Error 1
```

Which I think has to do with some newer changes made to SFML that haven't been accounted for with the static commands. But I'm not sure about this, and also don't know much about statically linked binaries on linux.

Reply:
Oh well. That's unfortunate, but this is probably still and improvement. I'll have a look at why this might happen when I get around to it, but I don't think it's important for this pull request.